database-commons/src/main/java/io/cdap/plugin/db/CommonSchemaReader.java

```diff
@@ -61,4 +63,8 @@ public Schema getSchema(ResultSetMetaData metadata, int index) throws SQLExcepti
   public boolean shouldIgnoreColumn(ResultSetMetaData metadata, int index) throws SQLException {
     return false;
   }
+
+  public void setSessionID(String sessionID) {
```

Comment:
try to avoid setters when possible. They often violate encapsulation, as outside classes are now able to change the internal state of this class. Normally, it's better to pass in the value into the constructor as save it as a final variable.